package.json

```diff
@@ -27,7 +27,7 @@
     "@angular/router": "^14.2.12",
     "@laserfiche/lf-js-utils": "^4.0.3",
     "@laserfiche/lf-repository-api-client": "^1.1.3",
-    "@laserfiche/lf-ui-components-services": "^14.0.0",
+    "@laserfiche/lf-ui-components-services": "^14.1.0--preview-4721943394",
```

Comment:
use production package once release

Reply:
we still need to replace swap out the jsdeliver url in index.html with the lfxstatic url